apps/studio/package.json

```diff
@@ -29,6 +29,8 @@
     "@aws-sdk/client-redshift": "^3.145.0",
     "@azure/msal-node": "^2.12.0",
     "@electron/remote": "^2.0.10",
+    "@cleverbrush/knex-clickhouse": "^1.1.10",
```

Comment:
This adds knex-clickhouse, but you also have `knex-clickhouse` in the source tree. Do we need both?